registry/handlers/context.go

```diff
@@ -25,6 +25,9 @@ type Context struct {
 	// should be scoped to a single repository. This field may be nil.
 	Repository distribution.Repository
 
+	// RepositoryRemover provides method to delete a repository
+	RepositoryRemover distribution.RepositoryRemover
```

Comment:
Where does this get read from?

Reply:
This is supposed to be read by delete endpoint implementations which are not there in OSS but is implemented in Hub fork. LMK if this seems awkward. I am fine with not storing it at all and having 
```go
			context.Repository, _ = notifications.Listen(
				repository,
				context.App.repoRemover,
				app.eventBridge(context, r))
```
instead.

Reply:
@dmcgowan Do you think it would be ok for registry to implement DELETE endpoints? `DELETE /v2/repo/reponame` and `DELETE /v2/repo/tags/tagname`. They need not be in OCI but it doesn't hurt having an implementation. I bet people are manually deleting content from storage directly anyways.